src/node/rpc/member_frontend.h

```diff
@@ -609,6 +610,19 @@ namespace ccf
         .set_auto_schema<StateDigest, void>()
         .install();
 
+      auto test = [this](ccf::endpoints::EndpointContext& ctx) {
+        const auto& caller_identity =
+          ctx.template get_caller<ccf::MemberCOSESign1AuthnIdentity>();
+
+        ctx.rpc_ctx->set_response_header(
+          http::headers::CONTENT_TYPE, http::headervalues::contenttype::JSON);
+        ctx.rpc_ctx->set_response_body(nlohmann::json().dump());
+        ctx.rpc_ctx->set_response_status(HTTP_STATUS_OK);
+      };
+      make_endpoint("/test", HTTP_POST, test, member_sig_only)
```

Comment:
nit: rename the endpoint name?

Reply:
This is only temporary, I am not really going to add an endpoint to governance just for testing.